cli/tests/integrations/common.py

```diff
@@ -630,7 +630,7 @@ def ssh_output(cmd):
     proc, master = popen_tty(cmd)
 
     # wait for the ssh connection
-    time.sleep(3)
+    time.sleep(5)
```

Comment:
can we do some kind of loop to test for the connection instead of just increasing the sleep?